packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -270,26 +270,30 @@ <h2 class="demo-title">Change Material Metalness and Roughness Factors</h2>
 
 modelViewerParameters.addEventListener("load", (ev) => {
 
-  let material = modelViewerParameters.model.materials[0];
-
-  let metalnessDisplay = document.querySelector("#metalness-value");
-  let roughnessDisplay = document.querySelector("#roughness-value");
-
-  metalnessDisplay.textContent = material.pbrMetallicRoughness.metallicFactor;
-  roughnessDisplay.textContent = material.pbrMetallicRoughness.roughnessFactor;
-
-  // Defaults to gold
-  material.pbrMetallicRoughness.setBaseColorFactor([0.7294, 0.5333, 0.0392]);
-
-  document.querySelector('#metalness').addEventListener('input', (event) => {
-    material.pbrMetallicRoughness.setMetallicFactor(event.target.value);
-    metalnessDisplay.textContent = event.target.value;
-  });
-
-  document.querySelector('#roughness').addEventListener('input', (event) => {
-    material.pbrMetallicRoughness.setRoughnessFactor(event.target.value);
-    roughnessDisplay.textContent = event.target.value;
-  });
+  // let material = modelViewerParameters.model.materials[0];
+
+  // let metalnessDisplay = document.querySelector("#metalness-value");
+  // let roughnessDisplay = document.querySelector("#roughness-value");
+
+  // metalnessDisplay.textContent = material.pbrMetallicRoughness.metallicFactor;
+  // roughnessDisplay.textContent = material.pbrMetallicRoughness.roughnessFactor;
+
+  // // Defaults to gold
+  // material.pbrMetallicRoughness.setBaseColorFactor([0,0,1]);
+  // material.pbrMetallicRoughness.setMetallicFactor(0);
+  // material.pbrMetallicRoughness.setRoughnessFactor(1);
+  // material.setEmissiveFactor([0, 0, 1]);
+  // document.querySelector('#metalness').addEventListener('input', (event) => {
+  //   material.pbrMetallicRoughness.setMetallicFactor(event.target.value);
+  //   metalnessDisplay.textContent = event.target.value;
+  // });
+
+  // document.querySelector('#roughness').addEventListener('input', (event) => {
+  //   //material.pbrMetallicRoughness.setRoughnessFactor(event.target.value);
+  //   // material.setEmissiveFactor([event.target.value, 0, event.target.value]);
+  //   //material.pbrMetallicRoughness.setRoughnessFactor(event.target.value);
+  //   roughnessDisplay.textContent = event.target.value * 5;
+  // });
```

Comment:
Perhaps copy this into a new example instead? You'll need to add an entry in examples.json.

Reply:
If this becomes part of the API then as a separate mixin it would obviously require separate examples, I just set this for minimal testing during dev